lms/djangoapps/learner_home/serializers.py

```diff
@@ -502,12 +505,12 @@ def get_courseProvider(self, entitlement):
 
 
 class SuggestedCourseSerializer(serializers.Serializer):
-    """Serializer for a suggested course"""
+    """Serializer for a suggested course from recommendation engine"""
 
-    bannerUrl = serializers.URLField()
-    logoUrl = serializers.URLField()
+    bannerUrl = serializers.URLField(source="logo_image_url")
```

Comment:
This was the design decision? What did this end up being?

Reply:
@jansenk, the decision was to use the org logo for the banner and leave out the spot that was previously being used for the org logo